cs4660/graph/graph.py

```diff
@@ -86,22 +105,59 @@ def __init__(self):
         self.adjacency_list = {}
 
     def adjacent(self, node_1, node_2):
-        pass
+        if node_1 not in self.adjacency_list:
+            return False
+
+        return node_2 in self.adjacency_list[node_1]
```

Comment:
the data structure used here is a bit off. It misses the capability to store the edge weight which then will be wrong on homework 2 (think of Dijkstra Search and A* Search).